### ISSUE TITLE
Original video number should be provided when downloading the entire playlist using youtube-dl

### ISSUE THREAD

**Issue by babygame0ver** [rg3/youtube-dl#15999]
## Please follow the guide below

- You will be asked some questions and requested to provide some information, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *issue* (like this: `[x]`)
- Use the *Preview* tab to see what your issue will actually look like

---

### Make sure you are using the *latest* version: run `youtube-dl --version` and ensure your version is *2018.03.26.1*. If it's not, read [this FAQ entry](https://github.com/rg3/youtube-dl/blob/master/README.md#how-do-i-update-youtube-dl) and update. Issues with outdated version will be rejected.
- [x ] I've **verified** and **I assure** that I'm running youtube-dl **2018.03.26.1**

### Before submitting an *issue* make sure you have:
- [ ] At least skimmed through the [README](https://github.com/rg3/youtube-dl/blob/master/README.md), **most notably** the [FAQ](https://github.com/rg3/youtube-dl#faq) and [BUGS](https://github.com/rg3/youtube-dl#bugs) sections
- [ ] [Searched](https://github.com/rg3/youtube-dl/search?type=Issues) the bugtracker for similar issues including closed ones
- [ ] Checked that provided video/audio/playlist URLs (if any) are alive and playable in a browser

### What is the purpose of your *issue*?
- [ ] Bug report (encountered problems with youtube-dl)
- [ ] Site support request (request for adding support for a new site)
- [x ] Feature request (request for a new functionality)
- [ ] Question
- [ ] Other

---

I want a new functionality, which I think it will help the other users also who are using youtube-dl. 
I am using youtube-dl for a year now. It's a best tool for programmer. But I want to say this from 3-4 months but I am so lazy , but Today I realized that I should suggest this feature to the newer version in the future. I was downloading a playlist with having 500 videos in it. It was downloaded successfully but there is not a number in the file name which is the biggest problem when I am downloading the entire playlist as I have to go back to YouTube, Then search back the playlist and then check which one to watch next. So please add this feature : **that the number of the video should having the original number as it has on the YouTube and number shouldn't change if user start the playlist from certain point like giving the argument of --playlist-start to the tool.** 
Please add this feature specially when a user enter the playlist to download or ask the user to prompt whether they want to save the name with the number or not for the destination file.
Thank you

**Comment by dstftw** [comment]
Video id is in filename by default.

**Comment by babygame0ver** [comment]
Open this url https://www.youtube.com/playlist?list=PLBlnK6fEyqRjMH3mWf6kwqiTbT798eAOm , 
If you will download all the videos using youtube-dl https://www.youtube.com/playlist?list=PLBlnK6fEyqRjMH3mWf6kwqiTbT798eAOm 
It will start downloading the video files. I have downloaded all the videos and now there s no id as defined by the author in the videos. So It's about 200 videos and whenever I have to study I have to get back to the playlist on the YouTube and then rename them according to the number.
All videos should have naming like this **original-number_video-name**  when it is downloaded. Think of a scenario where I don't have the internet connection and I have to see which video I have watched last. I think it is more clear now.
